api/src/functions/auth.js

```diff
@@ -18,7 +19,13 @@ export const handler = async (event, context) => {
     // address in a toast message so the user will know it worked and where
     // to look for the email.
     handler: (user) => {
-      return user
+      const res = sendEmail({
```

Comment:
Since we're returning the response here as `res` that the client uses on the page, we should probably await this:
```js
    handler: async (user) => {
      const res = await sendEmail({
      // ...
```

Reply:
Yeah you're right